app/src/main/AndroidManifest.xml

```diff
@@ -26,7 +26,7 @@
             android:theme="@style/Theme.AppCompat.NoActionBar" />
 
         <activity
-            android:name=".ui.MainActivity"
+            android:name=".ui.AppActivity"
```

Comment:
What was the reason behind this?

Reply:
imo it didn't convey what it actually did
Main of what

Reply:
Dunno, app? Honestly, doesn't matter. MainActivity is more of a convention for Android devs, such as mFieldname for private java fields ig

Reply:
ah i remembered why exactly i did
cause MainDestinations sounds like it could be from any activity
since its side by side with other destinations

Reply:
ah, okay then